index.html

```diff
@@ -171,6 +171,11 @@ <h1 style="margin-bottom: 7px;">Text to Handwriting</h1>
         </div>
         <div class="col output-container">
           <div id="output" class="output"></div>
+          <div style="text-align: center;padding-top: 20px;display: none;" id="nav">
+            <span class="blue-button" id="page-nav-left">&#60;&#8722;</span>
```

Comment:
Will it break anything if we turn it into `<button>`? Currently, these buttons would be inaccessible to the people using the keyboard tab to navigate

Reply:
No, it will not break anything on the site. I have made the changes and tested it and it works fine !